cdn/dev/css/template.css

```diff
@@ -1099,6 +1099,10 @@ body[data-device |= Windows] #cta-big-Windows{
 	display: block;
 }
 
+body[data-device |= Linux] #cta-big-Linux{
+	display: block;
+}
+
```

Comment:
Is this all that is needed to add support for Linux, or did I miss something (I didn't know how to test this locally)?

Reply:
Checked here -- looks good